drivers/net/ethernet/renesas/Kconfig

```diff
@@ -54,6 +54,15 @@ config RENESAS_ETHER_SWITCH
 	  This driver supports the following SoCs:
 		- R8A779Fx.
 
+config RENESAS_FWD_MON
+	tristate "Renesas forward monitoring network device support"
```

Comment:
AFAIK, `tistate` means that this feature can be built as a module, which is not true, as it is not implemented in a separate .c file

Also, do we really need separate config option for this feature?

Reply:
The redundant config was removed.